### ISSUE TITLE
Question regarding fine-tuning phase of ImageNet classification task

### ISSUE THREAD

**Issue by SohamTamba** [gidariss/FeatureLearningRotNet#13]
The paper published [here](https://arxiv.org/pdf/1803.07728.pdf) explains how the pretext task training is conducted, but not how the transfer learning is conducted. I had some questions regarding the procedure for transfer learning for the ImageNet classification task.

The entire procedure can be described as:
a) Train an AlexNet using the rotation prediction pretext task on the entire ImageNet dataset.
b) Freeze all layers except the fully connected layers.
c) Train the AlexNet using the Imagenet dataset using the ImageNet labels.

1) During phase (c), is the entire imagenet dataset used? Or is a fraction of it used? I would expect self-supervised learning to fine-tune using a relatively small dataset.
2) What hyper-parameters are used during phase (c)? Such as learning rate, weight decay etc.

Thank you.

**Comment by MrChenFeng** [comment]
Same question here. If all labels used in the transfer learning period, the performance of self-supervision become a little not obvious, from a relastic view. Considering the motivation of self-supervised learning is the lack of labels.
